cmake/LinuxPackaging.cmake

```diff
@@ -48,6 +48,9 @@ if(DEB_DETECT_DEPENDENCIES AND DPKG_CMD AND DPKGQ_CMD)
 	if(WITH_XML_BACKEND)
 		set(PACKAGES "${PACKAGES} libxml2")
 	endif()
+	if(WITH_SERIAL_BACKEND)
```

Comment:
these packaging changes look like they could go into a separate commit that handles only Linux packaging